docs/setup/config/sip-profile.md

```diff
@@ -12,14 +12,18 @@ When someone logs into the [Web Phone](https://docs.connexcs.com/webphone/), the
 
 ## Steps to setup the SIP Profile
 
-1. Go to Setup :material-menu-right: Config :material-menu-right: SIP Profile and you will see a screen as shown below. Then click the `+` button.![sip12](/setup/img/sip12.jpg)
+1. Go to **Setup :material-menu-right: Config :material-menu-right: SIP Profile** and you will see a screen as shown below. Then click the **blue `+` button**.<img src="/setup/img/sippro.png">
 
-2. A window will appear after you click on the `+` button. It will let you create a new SIP Profile. As you can see, the window has 3 fields for setting up the SIP Profile.![sip2](/setup/img/sip2.jpg)
+2. A window will appear after you click on the `+` button. It will let you create a new SIP Profile. Fill in the following details.
 
-    2.1 **Name**: This is a required field that will display your SIP Profile's name.
+      + **Name**: This is a required field that will display your SIP Profile's name.
+      + **Profile Capacity**: This field enables you to add the number of people who can use this profile to access the features.
+      + **Flags**: This field allows you to select one or multiple features that the user can access.![sip3](/setup/img/sip3.jpg)
+      + **Barge Group**: This field allows you add a group to the user. It allows the user to `listen` to the conversation of the specified group using Barge in [ConneXML-Barge Groups](https://docs.connexcs.com/class5/connexml/#dial).
```

Comment:
This feature allows you to add a group to a user. The user can then listen to conversations within the specified group using the Barge functionality in [ConneXML-Barge Groups].

Reply:
done